packages/react-native-usb/ios/ReactNativeUsbModule.swift

```diff
@@ -0,0 +1,37 @@
+import ExpoModulesCore
+
+// This file is mostly empty because we doesn't support iOS/Mac (yet).
+
+public class ReactNativeUsbModule: Module {
+  // Each module class must implement the definition function. The definition consists of components
+  // that describes the module's functionality and behavior.
+  // See https://docs.expo.dev/modules/module-api for more details about available components.
+  public func definition() -> ModuleDefinition {
+    // Sets the name of the module that JavaScript code will use to refer to the module. Takes a string as an argument.
+    // Can be inferred from module's class name, but it's recommended to set it explicitly for clarity.
+    // The module will be accessible from `requireNativeModule('ReactNativeUsb')` in JavaScript.
+    Name("ReactNativeUsb")
+
+    // Sets constant properties on the module. Can take a dictionary or a closure that returns a dictionary.
```

Comment:
Maybe it would be nice to remove all commented code for the information clarity

Reply:
I want to keep it, will be easier to pick up with examles if someone decide to do iOS